OrbitSsh/Socket.cpp

```diff
@@ -4,6 +4,17 @@
 
 #include "OrbitSsh/Socket.h"
 
+#ifndef _WIN32
```

Comment:
Again, I'd switch the cases and have a `ifdef` instead.

Reply:
Good point. Done.